types/sauronjs/core/service.d.ts

```diff
@@ -0,0 +1,7 @@
+export class Service {
```

Comment:
Same thing here - you should `declare` the class and then `export = Service;`.

Reply:
Done.